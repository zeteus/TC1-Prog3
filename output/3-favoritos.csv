1;Podcast;31;Geek;47.57
1;Podcast;34;Atualidades;62.5
1;Podcast;99;Noticias;24.07
1;Música;1;Rock;4.17
1;Música;9;Rock;4.15
1;Música;11;Rock;4
1;Música;12;Rock;3.55
1;Música;49;Funk;2.32
1;Música;75;Eletrônica;3
1;Música;81;Hip-Hop;2.63
1;Música;85;Hip-Hop;4.08
2;Podcast;27;Geek;40.38
2;Podcast;36;Atualidades;56.35
2;Podcast;89;Pop;144.33
2;Podcast;99;Noticias;24.07
2;Música;1;Rock;4.17
2;Música;8;Rock;3.98
2;Música;15;Indie;3.48
2;Música;19;Indie;6.65
2;Música;44;Funk;2.85
2;Música;54;Sertanejo;3.33
2;Música;58;Sertanejo;2.5
2;Música;64;Sertanejo;2.7
2;Música;71;Eletrônica;3.23
2;Música;72;Eletrônica;2
2;Música;84;Hip-Hop;2.35
3;Podcast;26;Geek;131.73
3;Podcast;28;Geek;59.23
3;Podcast;29;Geek;42.45
3;Podcast;33;Geek;44.5
3;Podcast;38;Atualidades;50.22
3;Música;1;Rock;4.17
3;Música;11;Rock;4
3;Música;22;Indie;5.28
3;Música;44;Funk;2.85
3;Música;49;Funk;2.32
3;Música;53;Funk;2.88
3;Música;57;Sertanejo;3.1
3;Música;63;Sertanejo;2.47
3;Música;71;Eletrônica;3.23
3;Música;84;Hip-Hop;2.35
3;Música;85;Hip-Hop;4.08
3;Música;86;Hip-Hop;4.22
4;Podcast;29;Geek;42.45
4;Podcast;39;Atualidades;47.2
4;Podcast;96;Noticias;20.6
4;Música;3;Rock;3.8
4;Música;8;Rock;3.98
4;Música;14;Indie;4.77
4;Música;18;Indie;4.4
4;Música;21;Indie;5.7
4;Música;42;Funk;2.05
4;Música;46;Funk;2.37
4;Música;51;Funk;2.87
4;Música;56;Sertanejo;3.15
4;Música;69;Sertanejo;3.12
4;Música;70;Sertanejo;2.77
4;Música;78;Eletrônica;3.42
4;Música;83;Hip-Hop;2.47
5;Podcast;39;Atualidades;47.2
5;Podcast;40;Atualidades;65
5;Música;8;Rock;3.98
5;Música;10;Rock;3.53
5;Música;51;Funk;2.87
5;Música;55;Sertanejo;2.8
5;Música;73;Eletrônica;3.08
5;Música;79;Hip-Hop;2.62
6;Podcast;26;Geek;131.73
6;Podcast;40;Atualidades;65
6;Música;44;Funk;2.85
6;Música;56;Sertanejo;3.15
6;Música;61;Sertanejo;2.33
6;Música;67;Sertanejo;4.18
6;Música;87;Hip-Hop;5.62
7;Podcast;28;Geek;59.23
7;Podcast;33;Geek;44.5
7;Podcast;37;Atualidades;58.02
7;Podcast;97;Noticias;22.93
7;Música;5;Rock;3
7;Música;24;K-Pop;3.82
7;Música;46;Funk;2.37
7;Música;48;Funk;3.77
7;Música;52;Funk;4.2
7;Música;57;Sertanejo;3.1
7;Música;62;Sertanejo;3.08
7;Música;79;Hip-Hop;2.62
7;Música;85;Hip-Hop;4.08
8;Podcast;31;Geek;47.57
8;Podcast;33;Geek;44.5
8;Podcast;92;Pop;113.23
8;Podcast;93;Pop;101.7
8;Podcast;99;Noticias;24.07
8;Música;50;Funk;3.78
8;Música;61;Sertanejo;2.33
8;Música;68;Sertanejo;3.13
8;Música;80;Hip-Hop;1.9
8;Música;85;Hip-Hop;4.08
9;Podcast;26;Geek;131.73
9;Podcast;34;Atualidades;62.5
9;Podcast;98;Noticias;21.27
9;Música;2;Rock;2.62
9;Música;6;Rock;4.78
9;Música;9;Rock;4.15
9;Música;12;Rock;3.55
9;Música;70;Sertanejo;2.77
10;Podcast;92;Pop;113.23
10;Música;11;Rock;4
10;Música;20;Indie;4.18
10;Música;43;Funk;2.63
10;Música;44;Funk;2.85
10;Música;53;Funk;2.88
10;Música;64;Sertanejo;2.7
10;Música;72;Eletrônica;2
11;Podcast;29;Geek;42.45
11;Música;12;Rock;3.55
11;Música;24;K-Pop;3.82
11;Música;42;Funk;2.05
11;Música;45;Funk;2.43
11;Música;47;Funk;2.97
11;Música;48;Funk;3.77
11;Música;75;Eletrônica;3
12;Podcast;32;Geek;79.5
12;Podcast;36;Atualidades;56.35
12;Música;5;Rock;3
12;Música;45;Funk;2.43
12;Música;50;Funk;3.78
12;Música;75;Eletrônica;3
12;Música;77;Eletrônica;2.62
12;Música;86;Hip-Hop;4.22
14;Podcast;27;Geek;40.38
14;Podcast;28;Geek;59.23
14;Podcast;35;Atualidades;63.42
14;Podcast;94;Pop;107.33
14;Música;1;Rock;4.17
14;Música;5;Rock;3
14;Música;15;Indie;3.48
14;Música;21;Indie;5.7
14;Música;43;Funk;2.63
14;Música;63;Sertanejo;2.47
14;Música;67;Sertanejo;4.18
14;Música;78;Eletrônica;3.42
14;Música;85;Hip-Hop;4.08
15;Podcast;27;Geek;40.38
15;Podcast;34;Atualidades;62.5
15;Podcast;36;Atualidades;56.35
15;Podcast;40;Atualidades;65
15;Podcast;92;Pop;113.23
15;Podcast;99;Noticias;24.07
15;Música;5;Rock;3
15;Música;76;Eletrônica;4.62
16;Podcast;33;Geek;44.5
16;Podcast;38;Atualidades;50.22
16;Podcast;39;Atualidades;47.2
16;Música;54;Sertanejo;3.33
16;Música;72;Eletrônica;2
16;Música;74;Eletrônica;3.13
16;Música;86;Hip-Hop;4.22
17;Podcast;91;Pop;151.8
17;Música;17;Indie;4.13
17;Música;21;Indie;5.7
17;Música;68;Sertanejo;3.13
17;Música;70;Sertanejo;2.77
17;Música;72;Eletrônica;2
17;Música;78;Eletrônica;3.42
18;Podcast;27;Geek;40.38
18;Podcast;93;Pop;101.7
18;Podcast;94;Pop;107.33
18;Música;9;Rock;4.15
18;Música;11;Rock;4
18;Música;12;Rock;3.55
18;Música;42;Funk;2.05
18;Música;64;Sertanejo;2.7
18;Música;65;Sertanejo;2.83
18;Música;68;Sertanejo;3.13
18;Música;69;Sertanejo;3.12
19;Podcast;98;Noticias;21.27
19;Música;11;Rock;4
19;Música;14;Indie;4.77
19;Música;46;Funk;2.37
19;Música;53;Funk;2.88
19;Música;54;Sertanejo;3.33
19;Música;55;Sertanejo;2.8
19;Música;56;Sertanejo;3.15
19;Música;61;Sertanejo;2.33
19;Música;77;Eletrônica;2.62
19;Música;79;Hip-Hop;2.62
19;Música;81;Hip-Hop;2.63
19;Música;82;Hip-Hop;2.18
20;Podcast;31;Geek;47.57
20;Podcast;94;Pop;107.33
20;Música;8;Rock;3.98
20;Música;18;Indie;4.4
20;Música;24;K-Pop;3.82
20;Música;55;Sertanejo;2.8
20;Música;59;Sertanejo;2.3
21;Podcast;39;Atualidades;47.2
21;Podcast;90;Pop;141.02
21;Podcast;95;Pop;120.77
21;Podcast;98;Noticias;21.27
21;Música;1;Rock;4.17
21;Música;21;Indie;5.7
21;Música;41;Funk;2.18
21;Música;44;Funk;2.85
21;Música;56;Sertanejo;3.15
21;Música;66;Sertanejo;2.88
21;Música;71;Eletrônica;3.23
21;Música;74;Eletrônica;3.13
22;Podcast;32;Geek;79.5
22;Podcast;34;Atualidades;62.5
22;Podcast;92;Pop;113.23
22;Podcast;96;Noticias;20.6
22;Podcast;99;Noticias;24.07
22;Música;12;Rock;3.55
22;Música;25;K-Pop;3.58
22;Música;42;Funk;2.05
22;Música;45;Funk;2.43
22;Música;62;Sertanejo;3.08
22;Música;72;Eletrônica;2
23;Podcast;28;Geek;59.23
23;Música;12;Rock;3.55
23;Música;21;Indie;5.7
23;Música;22;Indie;5.28
23;Música;42;Funk;2.05
23;Música;43;Funk;2.63
23;Música;51;Funk;2.87
23;Música;54;Sertanejo;3.33
23;Música;61;Sertanejo;2.33
23;Música;81;Hip-Hop;2.63
23;Música;82;Hip-Hop;2.18
23;Música;86;Hip-Hop;4.22
24;Podcast;38;Atualidades;50.22
24;Podcast;39;Atualidades;47.2
24;Podcast;40;Atualidades;65
24;Podcast;100;Noticias;23.7
24;Música;6;Rock;4.78
24;Música;20;Indie;4.18
24;Música;21;Indie;5.7
24;Música;23;K-Pop;3.62
24;Música;41;Funk;2.18
24;Música;42;Funk;2.05
24;Música;46;Funk;2.37
24;Música;53;Funk;2.88
24;Música;69;Sertanejo;3.12
24;Música;73;Eletrônica;3.08
25;Podcast;26;Geek;131.73
25;Podcast;30;Geek;80.67
25;Podcast;34;Atualidades;62.5
25;Podcast;92;Pop;113.23
25;Podcast;94;Pop;107.33
25;Podcast;96;Noticias;20.6
25;Podcast;99;Noticias;24.07
25;Música;9;Rock;4.15
25;Música;23;K-Pop;3.62
25;Música;52;Funk;4.2
25;Música;71;Eletrônica;3.23
25;Música;82;Hip-Hop;2.18
25;Música;83;Hip-Hop;2.47
26;Podcast;28;Geek;59.23
26;Podcast;33;Geek;44.5
26;Podcast;37;Atualidades;58.02
26;Podcast;94;Pop;107.33
26;Música;6;Rock;4.78
26;Música;7;Rock;3.9
26;Música;14;Indie;4.77
26;Música;22;Indie;5.28
26;Música;56;Sertanejo;3.15
26;Música;59;Sertanejo;2.3
26;Música;63;Sertanejo;2.47
26;Música;65;Sertanejo;2.83
26;Música;67;Sertanejo;4.18
26;Música;68;Sertanejo;3.13
26;Música;74;Eletrônica;3.13
26;Música;80;Hip-Hop;1.9
26;Música;81;Hip-Hop;2.63
27;Música;19;Indie;6.65
28;Podcast;26;Geek;131.73
28;Podcast;28;Geek;59.23
28;Podcast;92;Pop;113.23
28;Música;1;Rock;4.17
28;Música;2;Rock;2.62
28;Música;5;Rock;3
28;Música;9;Rock;4.15
28;Música;14;Indie;4.77
28;Música;16;Indie;4.05
28;Música;23;K-Pop;3.62
28;Música;43;Funk;2.63
28;Música;47;Funk;2.97
28;Música;60;Sertanejo;2.7
28;Música;64;Sertanejo;2.7
28;Música;65;Sertanejo;2.83
28;Música;71;Eletrônica;3.23
28;Música;79;Hip-Hop;2.62
28;Música;84;Hip-Hop;2.35
28;Música;87;Hip-Hop;5.62
29;Podcast;33;Geek;44.5
29;Podcast;35;Atualidades;63.42
29;Podcast;38;Atualidades;50.22
29;Podcast;39;Atualidades;47.2
29;Música;16;Indie;4.05
29;Música;19;Indie;6.65
29;Música;45;Funk;2.43
29;Música;53;Funk;2.88
29;Música;58;Sertanejo;2.5
29;Música;62;Sertanejo;3.08
29;Música;68;Sertanejo;3.13
29;Música;81;Hip-Hop;2.63
