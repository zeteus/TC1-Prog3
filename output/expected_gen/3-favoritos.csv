Código Assinante;Tipo Mídia;Código Mídia;Gênero;Duração
2;M;45;Horror punk;3.9
3;P;193;Feminist;24.78
3;P;191;Comedy and humor;6.3
3;P;189;Talk;108.77
3;P;178;Science fiction;113.17
3;P;166;Business and finance;81.67
3;P;160;Science fiction;89.46
3;P;149;Video game;92.99
3;P;131;Music;81.54
3;P;119;History;28.74
3;P;116;Talk;95.68
3;P;90;Law;9.19
3;P;86;Automobile;40.5
3;P;50;Educational;46.78
3;P;46;Horror;93.16
3;P;30;Sports;73.8
3;P;25;Automobile;5.89
3;P;16;Science;44.26
3;M;172;Progressive bluegrass;9.55
3;M;162;Music drama;3
3;M;112;Waltz;6.1
3;M;38;Waltz;7.29
3;M;36;Music drama;8.82
3;M;31;Khplam wai;6.94
12;P;198;Arts;33.15
12;P;115;Technology;57.54
12;P;79;Comedy and humor;57.35
12;P;51;Political;8.35
12;P;50;Educational;46.78
12;P;18;Sports;44.43
12;M;192;Waltz;10.81
12;M;162;Music drama;3
12;M;145;Rare groove;8.5
12;M;111;Progressive bluegrass;4.41
12;M;76;Khyal;4.38
12;M;1;Progressive bluegrass;7.85
13;M;63;Waltz;8.89
13;M;57;Go-go;2.12
15;P;184;Advice;30.19
15;P;156;Film and television;71.19
15;P;136;History;76.69
15;P;105;Law;93.7
15;P;17;Educational;93.35
15;M;172;Progressive bluegrass;9.55
15;M;162;Music drama;3
15;M;126;Industrial rock;8.29
15;M;64;Jesus music;3.1
15;M;42;Waltz;8.65
15;M;38;Waltz;7.29
15;M;31;Khplam wai;6.94
25;P;185;Philosophy;40.4
25;P;184;Advice;30.19
25;P;170;Political;83.89
25;P;156;Film and television;71.19
25;P;109;Film and television;85.8
25;P;104;Talk;43.73
25;P;40;Comedy and humor;11.63
25;P;21;Science fiction;56.84
25;P;14;Educational;73.15
25;M;142;Swing music;7.13
25;M;103;Outsider music;1.34
25;M;102;Ars nova;8.11
25;M;82;Mexican son;10.97
25;M;48;Outsider music;8.2
25;M;27;Shoegaze;6.9
25;M;1;Progressive bluegrass;7.85
26;P;184;Advice;30.19
26;P;109;Film and television;85.8
26;P;92;History;117.79
26;M;190;Shoegaze;10.37
26;M;188;Jesus music;3.64
26;M;130;Horror punk;2.7
26;M;102;Ars nova;8.11
26;M;78;Outsider music;6.18
28;P;147;Religion and spirituality;92.5
28;P;143;Interview;17.83
28;P;141;Video game;112.22
28;P;92;History;117.79
28;P;87;Comedy and humor;9.94
28;P;86;Automobile;40.5
28;P;25;Automobile;5.89
28;P;14;Educational;73.15
28;P;4;Talk;64.97
28;M;192;Waltz;10.81
28;M;96;Progressive bluegrass;9.59
28;M;81;Dirty south;3.9
28;M;55;Khyal;5.82
32;P;135;Philosophy;38.95
32;P;107;Technology;47.77
32;P;99;History;32.19
32;P;94;Horror;56.5
32;P;51;Political;8.35
32;P;50;Educational;46.78
32;P;4;Talk;64.97
32;M;176;Music drama;5.75
32;M;144;Gamelan bebonangan;2.29
32;M;142;Swing music;7.13
32;M;81;Dirty south;3.9
32;M;61;Gunchei;5.63
32;M;33;Stoner rock;4.31
37;P;147;Religion and spirituality;92.5
37;P;80;Video game;81.43
37;P;47;Talk;113.61
37;P;19;Interview;114.51
37;P;10;Music;87.62
37;M;176;Music drama;5.75
37;M;145;Rare groove;8.5
37;M;130;Horror punk;2.7
37;M;108;La la;4.81
37;M;36;Music drama;8.82
38;P;124;Interview;98.5
38;P;79;Comedy and humor;57.35
38;P;17;Educational;93.35
38;M;190;Shoegaze;10.37
38;M;176;Music drama;5.75
38;M;167;Mor lam;8.16
38;M;125;La la;5.42
38;M;112;Waltz;6.1
38;M;102;Ars nova;8.11
38;M;77;Rare groove;10.49
38;M;29;Stoner rock;6.84
39;P;198;Arts;33.15
39;P;181;Sports;30.2
39;P;160;Science fiction;89.46
39;P;141;Video game;112.22
39;P;137;Sports;59.5
39;P;132;Film and television;74.18
39;P;128;Sports;23.12
39;P;73;Interview;30.76
39;P;50;Educational;46.78
39;P;47;Talk;113.61
39;P;17;Educational;93.35
39;P;10;Music;87.62
39;P;5;Automobile;98.15
39;M;194;Horror punk;4.34
39;M;152;Outsider music;2.85
39;M;111;Progressive bluegrass;4.41
39;M;95;Kaseko;7.17
39;M;82;Mexican son;10.97
39;M;78;Outsider music;6.18
39;M;77;Rare groove;10.49
39;M;72;Ars nova;5.24
39;M;71;Khplam wai;6.19
39;M;57;Go-go;2.12
39;M;42;Waltz;8.65
39;M;34;Dirty south;9.2
39;M;33;Stoner rock;4.31
39;M;26;Progressive bluegrass;8.59
40;P;179;LGBT-related;3.46
40;P;143;Interview;17.83
40;P;141;Video game;112.22
40;P;37;Religion and spirituality;76.36
40;P;25;Automobile;5.89
40;P;7;Talk;75.5
40;P;3;Sports;98.79
40;M;139;Industrial rock;6.29
40;M;112;Waltz;6.1
40;M;71;Khplam wai;6.19
40;M;59;Ars nova;2.76
40;M;53;Waltz;8.9
41;P;146;Music;46.37
41;P;136;History;76.69
41;P;132;Film and television;74.18
41;P;109;Film and television;85.8
41;P;99;History;32.19
41;P;56;Comedy and humor;61.46
41;P;52;Religion and spirituality;45.41
41;P;41;Feminist;6.93
41;P;30;Sports;73.8
41;P;14;Educational;73.15
41;P;2;Feminist;51.75
41;M;140;Swing music;10.87
41;M;95;Kaseko;7.17
41;M;82;Mexican son;10.97
41;M;77;Rare groove;10.49
41;M;72;Ars nova;5.24
41;M;29;Stoner rock;6.84
41;M;20;Seis;7.91
41;M;9;Progressive bluegrass;8.86
43;P;193;Feminist;24.78
43;P;99;History;32.19
43;P;87;Comedy and humor;9.94
43;M;150;Music drama;8.11
43;M;140;Swing music;10.87
44;P;185;Philosophy;40.4
44;P;156;Film and television;71.19
44;P;147;Religion and spirituality;92.5
44;P;128;Sports;23.12
44;P;115;Technology;57.54
44;P;32;Automobile;23.24
44;M;197;Outsider music;10.82
44;M;172;Progressive bluegrass;9.55
44;M;167;Mor lam;8.16
44;M;157;Outsider music;5.5
44;M;72;Ars nova;5.24
44;M;68;Mor lam;5.23
48;P;186;Technology;69.27
48;P;166;Business and finance;81.67
48;P;124;Interview;98.5
48;P;62;Science fiction;70.46
48;P;43;History;19.65
48;P;28;Technology;2.55
48;P;25;Automobile;5.89
48;P;24;Automobile;4.57
48;P;16;Science;44.26
48;M;188;Jesus music;3.64
48;M;172;Progressive bluegrass;9.55
48;M;153;Seis;8.3
48;M;125;La la;5.42
48;M;120;Dirty south;4.84
48;M;63;Waltz;8.89
48;M;57;Go-go;2.12
48;M;12;Outsider music;10.73

