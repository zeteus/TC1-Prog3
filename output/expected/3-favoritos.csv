CódigoAssinante;Tipo Mídia;Código Mídia;Gênero;Duração
1;P;99;Noticias;24,07
1;P;34;Atualidades;62,5
1;P;31;Geek;47,57
1;M;85;Hip-Hop;4,08
1;M;81;Hip-Hop;2,63
1;M;75;Eletrônica;3
1;M;49;Funk;2,32
1;M;12;Rock;3,55
1;M;11;Rock;4
1;M;9;Rock;4,15
1;M;1;Rock;4,17
2;P;99;Noticias;24,07
2;P;89;Pop;144,33
2;P;36;Atualidades;56,35
2;P;27;Geek;40,38
2;M;84;Hip-Hop;2,35
2;M;72;Eletrônica;2
2;M;71;Eletrônica;3,23
2;M;64;Sertanejo;2,7
2;M;58;Sertanejo;2,5
2;M;54;Sertanejo;3,33
2;M;44;Funk;2,85
2;M;19;Indie;6,65
2;M;15;Indie;3,48
2;M;8;Rock;3,98
2;M;1;Rock;4,17
3;P;38;Atualidades;50,22
3;P;33;Geek;44,5
3;P;29;Geek;42,45
3;P;28;Geek;59,23
3;P;26;Geek;131,73
3;M;86;Hip-Hop;4,22
3;M;85;Hip-Hop;4,08
3;M;84;Hip-Hop;2,35
3;M;71;Eletrônica;3,23
3;M;63;Sertanejo;2,47
3;M;57;Sertanejo;3,1
3;M;53;Funk;2,88
3;M;49;Funk;2,32
3;M;44;Funk;2,85
3;M;22;Indie;5,28
3;M;11;Rock;4
3;M;1;Rock;4,17
4;P;96;Noticias;20,6
4;P;39;Atualidades;47,2
4;P;29;Geek;42,45
4;M;83;Hip-Hop;2,47
4;M;78;Eletrônica;3,42
4;M;70;Sertanejo;2,77
4;M;69;Sertanejo;3,12
4;M;56;Sertanejo;3,15
4;M;51;Funk;2,87
4;M;46;Funk;2,37
4;M;42;Funk;2,05
4;M;42;Funk;2,05
4;M;21;Indie;5,7
4;M;18;Indie;4,4
4;M;14;Indie;4,77
4;M;8;Rock;3,98
4;M;3;Rock;3,8
5;P;40;Atualidades;65
5;P;39;Atualidades;47,2
5;M;79;Hip-Hop;2,62
5;M;73;Eletrônica;3,08
5;M;55;Sertanejo;2,8
5;M;51;Funk;2,87
5;M;10;Rock;3,53
5;M;8;Rock;3,98
6;P;40;Atualidades;65
6;P;26;Geek;131,73
6;M;87;Hip-Hop;5,62
6;M;67;Sertanejo;4,18
6;M;61;Sertanejo;2,33
6;M;56;Sertanejo;3,15
6;M;56;Sertanejo;3,15
6;M;44;Funk;2,85
7;P;97;Noticias;22,93
7;P;37;Atualidades;58,02
7;P;33;Geek;44,5
7;P;28;Geek;59,23
7;M;85;Hip-Hop;4,08
7;M;79;Hip-Hop;2,62
7;M;62;Sertanejo;3,08
7;M;57;Sertanejo;3,1
7;M;52;Funk;4,2
7;M;48;Funk;3,77
7;M;46;Funk;2,37
7;M;24;HP;3,82
7;M;5;Rock;3
8;P;99;Noticias;24,07
8;P;93;Pop;101,7
8;P;92;Pop;113,23
8;P;33;Geek;44,5
8;P;31;Geek;47,57
8;M;85;Hip-Hop;4,08
8;M;80;Hip-Hop;1,9
8;M;68;Sertanejo;3,13
8;M;61;Sertanejo;2,33
8;M;50;Funk;3,78
9;P;98;Noticias;21,27
9;P;34;Atualidades;62,5
9;P;26;Geek;131,73
9;M;70;Sertanejo;2,77
9;M;12;Rock;3,55
9;M;9;Rock;4,15
9;M;6;Rock;4,78
9;M;2;Rock;2,62
10;P;92;Pop;113,23
10;M;72;Eletrônica;2
10;M;64;Sertanejo;2,7
10;M;53;Funk;2,88
10;M;44;Funk;2,85
10;M;43;Funk;2,63
10;M;20;Indie;4,18
10;M;11;Rock;4
11;P;29;Geek;42,45
11;M;75;Eletrônica;3
11;M;48;Funk;3,77
11;M;47;Funk;2,97
11;M;45;Funk;2,43
11;M;42;Funk;2,05
11;M;24;K-Pop;3,82
11;M;12;Rock;3,55
12;P;36;Atualidades;56,35
12;P;32;Geek;79,5
12;M;86;Hip-Hop;4,22
12;M;77;Eletrônica;2,62
12;M;75;Eletrônica;3
12;M;50;Funk;3,78
12;M;45;Funk;2,43
12;M;5;Rock;3
14;P;94;Pop;107,33
14;P;35;Atualidades;63,42
14;P;35;Atualidades;63,42
14;P;28;Geek;59,23
14;P;27;Geek;40,38
14;M;85;Hip-Hop;4,08
14;M;78;Eletrônica;3,42
14;M;67;Sertanejo;4,18
14;M;63;Sertanejo;2,47
14;M;43;Funk;2,63
14;M;21;Indie;5,7
14;M;15;Indie;3,48
14;M;5;Rock;3
14;M;1;Rock;4,17
15;P;99;Noticias;24,07
15;P;99;Noticias;24,07
15;P;92;Pop;113,23
15;P;40;Atualidades;65
15;P;36;Atualidades;56,35
15;P;34;Atualidades;62,5
15;P;27;Geek;40,38
15;M;76;Eletrônica;4,62
15;M;5;Rock;3
16;P;39;Atualidades;47,2
16;P;38;Atualidades;50,22
16;P;33;Geek;44,5
16;M;86;Hip-Hop;4,22
16;M;74;Eletrônica;3,13
16;M;72;Eletrônica;2
16;M;54;Sertanejo;3,33
17;P;91;Pop;151,8
17;M;78;Eletrônica;3,42
17;M;72;Eletrônica;2
17;M;70;Sertanejo;2,77
17;M;68;Sertanejo;3,13
17;M;21;Indie;5,7
17;M;17;Indie;4,13
18;P;94;Pop;107,33
18;P;93;Pop;101,7
18;P;27;Geek;40,38
18;M;69;Sertanejo;3,12
18;M;68;Sertanejo;3,13
18;M;65;Sertanejo;2,83
18;M;64;Sertanejo;2,7
18;M;42;Funk;2,05
18;M;12;Rock;3,55
18;M;11;Rock;4
18;M;9;Rock;4,15
19;P;98;Noticias;21,27
19;M;82;Hip-Hop;2,18
19;M;81;Hip-Hop;2,63
19;M;79;Eletrônica;2,62
19;M;77;Eletrônica;2,62
19;M;61;Sertanejo;2,33
19;M;56;Sertanejo;3,15
19;M;55;Sertanejo;2,8
19;M;54;Sertanejo;3,33
19;M;53;Funk;2,88
19;M;46;Funk;2,37
19;M;46;Funk;2,37
19;M;14;Indie;4,77
19;M;11;Rock;4
20;P;94;Pop;107,33
20;P;31;Geek;47,57
20;M;59;Sertanejo;2,3
20;M;55;Sertanejo;2,8
20;M;24;K-Pop;3,82
20;M;18;Indie;4,4
20;M;8;Rock;3,98
21;P;98;Noticias;21,27
21;P;95;Pop;120,77
21;P;90;Pop;141,02
21;P;39;Atualidades;47,2
21;M;74;Eletrônica;3,13
21;M;71;Eletrônica;3,23
21;M;66;Sertanejo;2,88
21;M;56;Sertanejo;3,15
21;M;44;Funk;2,85
21;M;41;Funk;2,18
21;M;21;Indie;5,7
21;M;1;Rock;4,17
22;P;99;Noticias;24,07
22;P;96;Noticias;20,6
22;P;96;Noticias;20,6
22;P;92;Pop;113,23
22;P;34;Atualidades;62,5
22;P;32;Geek;79,5
22;M;72;Eletrônica;2
22;M;62;Sertanejo;3,08
22;M;45;Funk;2,43
22;M;42;Funk;2,05
22;M;25;K-Pop;3,58
22;M;12;Rock;3,55
23;P;28;Geek;59,23
23;M;86;Hip-Hop;4,22
23;M;82;Hip-Hop;2,47
23;M;81;Hip-Hop;2,63
23;M;61;Sertanejo;2,33
23;M;54;Sertanejo;3,33
23;M;51;Funk;2,87
23;M;43;Funk;2,63
23;M;42;Funk;2,05
23;M;22;Indie;5,28
23;M;21;Indie;5,7
23;M;12;Rock;3,55
24;P;100;Noticias;23,7
24;P;40;Atualidades;65
24;P;39;Atualidades;47,2
24;P;38;Atualidades;50,22
24;M;73;Eletrônica;3,08
24;M;69;Sertanejo;3,12
24;M;53;Funk;2,88
24;M;46;Funk;2,37
24;M;42;Funk;2,05
24;M;41;Funk;2,18
24;M;23;K-Pop;3,62
24;M;21;Indie;5,7
24;M;20;Indie;4,18
24;M;6;Rock;4,78
25;P;99;Noticias;24,07
25;P;96;Noticias;20,6
25;P;94;Pop;107,33
25;P;92;Pop;113,23
25;P;34;Atualidades;62,5
25;P;30;Geek;80,67
25;P;26;Geek;131,73
25;M;83;Hip-Hop;2,18
25;M;83;Hip-Hop;2,47
25;M;82;Hip-Hop;2,18
25;M;71;Eletrônica;3,23
25;M;52;Funk;4,2
25;M;23;K-Pop;3,62
25;M;9;Rock;4,15
26;P;94;Pop;107,33
26;P;37;Atualidades;58,02
26;P;33;Geek;44,5
26;P;28;Geek;59,23
26;M;81;Hip-Hop;2,63
26;M;80;Hip-Hop;1,9
26;M;74;Eletrônica;3,13
26;M;68;Sertanejo;3,13
26;M;68;Sertanejo;3,13
26;M;67;Sertanejo;4,18
26;M;67;Sertanejo;4,18
26;M;65;Sertanejo;2,83
26;M;63;Sertanejo;2,47
26;M;59;Sertanejo;2,3
26;M;56;Sertanejo;3,15
26;M;22;Indie;5,28
26;M;14;Indie;4,77
26;M;7;Rock;3,9
26;M;6;Rock;4,78
27;M;19;Indie;6,65
28;P;92;Pop;113,23
28;P;92;Pop;151,8
28;P;28;Geek;59,23
28;P;26;Geek;131,73
28;M;87;Hip-Hop;5,62
28;M;84;Hip-Hop;2,35
28;M;79;Hip-Hop;2,62
28;M;71;Eletrônica;3,23
28;M;65;Sertanejo;2,83
28;M;64;Sertanejo;2,7
28;M;60;Sertanejo;2,7
28;M;47;Funk;2,97
28;M;43;Funk;2,63
28;M;23;Indie;3,62
28;M;16;Indie;4,05
28;M;14;Indie;4,77
28;M;9;Rock;4,15
28;M;5;Rock;3
28;M;2;Rock;2,62
28;M;1;Rock;4,17
29;P;39;Atualidades;47,2
29;P;38;Atualidades;50,22
29;P;35;Atualidades;63,42
29;P;35;Atualidades;63,42
29;P;33;Geek;44,5
29;M;81;Hip-Hop;2,63
29;M;68;Sertanejo;3,13
29;M;62;Sertanejo;3,08
29;M;58;Sertanejo;2,5
29;M;53;Funk;2,88
29;M;45;Funk;2,43
29;M;19;Indie;6,65
29;M;16;Indie;4,05
